test-utils/runtime/src/lib.rs

```diff
@@ -1401,3 +1401,24 @@ mod tests {
 		runtime_api.test_witness(&block_id, proof, root).unwrap();
 	}
 }
+
+//#[derive(Clone)]
+//struct TestInherentDataProvider;
+//
+//const ERROR_TO_STRING: &str = "Found error!";
+//const TEST_INHERENT_0: InherentIdentifier = *b"testinh0";
+//
+//#[async_trait::async_trait]
+//impl sp_inherents::InherentDataProvider for TestInherentDataProvider {
```

Comment:
@bkchr This breaks with `1412 | impl sp_inherents::InherentDataProvider for TestInherentDataProvider {
       |                    ^^^^^^^^^^^^^^^^^^^^ not found in sp_inherents` but I don't understand why as `sp_inherents::InherentDataProvider` is there. Can you tell what I'm missing? 
       
 I want to make this work so we don't have this very same code in three different files.

Reply:
You need to guard the implementation with `#[cfg(feature = "std")]` as the trait is only available on std and not in wasm